### PR TITLE
Update routing.rst

### PR DESCRIPTION
**Pull request by ifdattic**
| Q | A |
| --- | --- |
| Doc fix? | yes |
| New docs? | no |
| Applies to | 2.3 |
| Fixed tickets |  |

### CODE REVIEW

**Comment by WouterJ** [comment]
I like it! :+1:

File: book/routing.rst [hunk 1]
```diff
@@ -1429,7 +1441,10 @@ Generating URLs with Query Strings
 The ``generate`` method takes an array of wildcard values to generate the URI.
 But if you pass extra ones, they will be added to the URI as a query string::
 
-    $this->get('router')->generate('blog', array('page' => 2, 'category' => 'Symfony'));
+    $this->get('router')->generate('blog', array(
+        'page' => 2,
+        'category' => 'Symfony'
```
Comment (javiereguiluz) [inline book/routing.rst]: A quick question: do we add a trailing comma for the last array element in the docs ... or do we only add it in code?

Comment (xabbuh) [inline book/routing.rst]: Usually we follow this rule in the docs too.

Comment (ifdattic) [inline book/routing.rst]: Crap, missed the comma, losing my sharpness I guess :D

Yea, usually docs have the trailing comma and as contribution docs has a section about having trailing comma, in my opinion it's a good idea to follow that rule in docs also.

**Pull request merged**